ArcSlider/ArcSliderBehaviorDecorator.js

```diff
@@ -0,0 +1,121 @@
+import hoc from '@enact/core/hoc';
+import platform from '@enact/core/platform';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {positionToAngle} from '../Arc/utils';
+import {angleToValue} from './utils';
+
+// Adds agate-specific arcSlider behaviors
+const ArcSliderBehaviorDecorator = hoc((config, Wrapped) => {
+	return class extends React.Component {
+		static displayName = 'ArcSliderBehaviorDecorator';
+
+		static propTypes = {
+			endAngle: PropTypes.number,
+			max: PropTypes.number,
+			min: PropTypes.number,
+			radius: PropTypes.number,
+			startAngle: PropTypes.number,
+			step: PropTypes.number,
+			strokeWidth: PropTypes.number
+		};
+
+		constructor (props) {
+			super(props);
+
+			this.componentRef = React.createRef();
+
+			this.state = {
+				value: props.min
+			};
+		}
+
+		onMouseDown = (ev) => {
```

Comment:
We usually name it with `handle` for event handlers. So it might be `handleMouseDown`.
As I mentioned before, it's better to use `onDown` event in Touchable, then it would be `handleDown`.